standalone/src/models/custom-application/custom-application-permission/presets/view-only-permissions.ts

```diff
@@ -2,15 +2,21 @@ import { entryPointUriPathToResourceAccesses } from '@commercetools-frontend/app
 import camelCase from 'lodash/camelCase';
 import sampleSize from 'lodash/sampleSize';
 import upperFirst from 'lodash/upperFirst';
-import { supportedViewOAuthScopes } from '../../custom-application-permission/constants';
-import CustomApplicationPermission from '../builder';
+import type { TBuilder } from '@/core';
+import { supportedViewOAuthScopes } from '../../constants';
+import { CustomApplicationPermissionGraphql } from '../index';
+import type { TCustomApplicationPermissionGraphql } from '../types';
 
-const preset = (entryPointUriPath = 'avengers', additionalPermission = '') => {
+const populatePreset = <TModel extends TCustomApplicationPermissionGraphql>(
```

Comment:
Same as before: this seems an unnecessary abstraction since we only have one preset in this file.